docs/external-oasis-repos.md

```diff
@@ -20,32 +20,32 @@ Hosting a repository on OASIS
 1.  If the respository ends in a new domain name that has not been distributed before, place a copy of the
     `domain.name.pub` public key from `domain.name` into `/srv/etc/keys` on both `oasis-replica` and
     `oasis-replica-itb`. If you do not have that key, ask the repository service representative how to obtain it.  In
-    order to support CVMFS client versions 2.2.X, a symbolic link of `domain.name.conf` has to be made in
+    order to support CVMFS client versions 2.2.X, make a symbolic link of `domain.name.conf`
     `/cvmfs/config-osg.opensciencegrid.org/etc/cvmfs/domain.d` pointing to `default.conf`. This symbolic link has to be
     created on the `oasis-itb` machine's copy of the `config-osg.opensciencegrid.org` repository and then copied to
     production with the `copy_config_osg` command on the oasis machine.
 
 1.  If the repository name matches `*.opensciencegrid.org` or `*.osgstorage.org`, respond to the ticket to ask the
-    administrator to continue with the next step; all other repositories (such as `*.egi.eu`) you are done.
+    administrator to continue with their next step and wait until the admin has updated the ticket to indicate that they
```

Comment:
I had to re-read that a few times to understand.  I think a comma after "their next step" would make it much clearer that the instruction "and wait" is for the GOC person and not the admin.